### PR TITLE
PLF-6913 URL link should be opened in a new tab

### PR DESCRIPTION
**Pull request by nttuyen**
Allow attribute target on a tag

### CODE REVIEW

**Comment by nttuyen** [comment]
PR updated by introduce new Filter to add target="_blank" on all link tag in activity before display it.

File: component/common/src/main/java/org/exoplatform/social/common/xmlprocessor/filters/OpenLinkNewTabFilterPlugin.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import org.exoplatform.social.common.xmlprocessor.DOMParser;
+import org.exoplatform.social.common.xmlprocessor.Tokenizer;
+import org.exoplatform.social.common.xmlprocessor.model.Node;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * @author <a href="mailto:tuyennt@exoplatform.com">Tuyen Nguyen The</a>.
+ */
```
Comment (thomasdelhomenie) [inline component/common/src/main/java/org/exoplatform/social/common/xmlprocessor/filters/OpenLinkNewTabFilterPlugin.java]: Please add a meaningful comment to explain what this plugin does and also comments on methods.

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/component-plugins-configuration.xml [hunk 2]
```diff
@@ -113,6 +113,7 @@
               <field name="tagAttributes">
                 <collection type="java.util.HashSet" item-type="java.lang.String">
                   <value><string>href</string></value>
+                  <value><string>target</string></value>
```
Comment (thomasdelhomenie) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/component-plugins-configuration.xml]: Is this change really needed ?

File: component/common/src/main/java/org/exoplatform/social/common/xmlprocessor/filters/OpenLinkNewTabFilterPlugin.java (hunk 1 above)
Comment (nttuyen) [inline component/common/src/main/java/org/exoplatform/social/common/xmlprocessor/filters/OpenLinkNewTabFilterPlugin.java]: The comment was added.

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/component-plugins-configuration.xml (hunk 2 above)
Comment (nttuyen) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/component-plugins-configuration.xml]: Removed

**Comment by thomasdelhomenie** [comment]
Merged
